python/cuspatial/cuspatial/tests/hausdorff_distance_test_toy.py

```diff
@@ -5,12 +5,12 @@
 
```

Comment:
Let's remove the word "toy" from all tests and filenames. Unit tests are not toys. :)

Also rename all pytests to match cudf pytest naming standard: `test_name.py` rather than `name_test.py` (better for lexicographical ordering).